packages/@aws-cdk/integ-runner/test/test-data/test-with-new-assets-diff.integ.snapshot/integ.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": "v1.0.0",
   "testCases": {
-    "xxxxx.test-with-new-assets": {
+    "integ.test-with-new-assets": {
```

Comment:
Regarding all those renames...  The `xxxxx` was done so that our integration test runner won’t pick them up, because those are not actually integration tests, but are only used for unit tests.

Would you mind naming them back? The new feature should be flexible enough the use any file name anyway.

Reply:
Sure, now that we can specify any prefix, this should be doable